integration_tests/rest_api/setup_data_hooks.py

```diff
@@ -183,6 +183,14 @@ def add_holding(txn):
             'targetQuantity': 1337
         })
 
+@hooks.before('/offers/{id}/accept > PATCH > 200 > application/json')
+def add_accept_info(txn):
+    patch_body(txn, {
+            'source': seeded_data['offer']['source'],
+            'sourceQuantity': seeded_data['offer']['sourceQuantity'],
```

Comment:
The API spec should be updated to match the way the protobuf now works, with a `count` of exchanges instead of a source quantity.